### PR TITLE
Parallellise CI tests for profiles

### PR DESCRIPTION
**Pull request by d4straub**
This is supposed to solve  #200

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -35,12 +35,29 @@ jobs:
...
-          nextflow run ${GITHUB_WORKSPACE} -profile test_multi,docker
-      - name: Run pipeline with ITS PacBio reads and Unite database
-        run: | 
-          nextflow run ${GITHUB_WORKSPACE} -profile test_pacbio_its,docker
-      - name: Run pipeline with double cutadapt steps on test data with double primers
+
+  profiles:
+    name: Run workflow profile
+    # Only run on push if this is the nf-core dev branch (merged PRs)
+    if: ${{ github.event_name != 'push' || (github.event_name == 'push' && github.repository == 'nf-core/mag') }}
```
Comment (emnilsson) [inline .github/workflows/ci.yml:42]: Is it supposed to be nf-core/mag?

Comment (d4straub) [reply .github/workflows/ci.yml:42]: uuups good catch, I copied it from mag, I'll fix it

Comment (d4straub) [reply .github/workflows/ci.yml]: Fixed

**Comment by emnilsson** [comment]
LGTM!

**Pull request merged**